src/main/kotlin/no/nav/familie/ef/personhendelse/client/pdl/PdlPersonRequest.kt

```diff
@@ -1,10 +1,14 @@
 package no.nav.familie.ef.personhendelse.client.pdl
 
-data class PdlPersonRequest<T>(val variables: T,
-                               val query: String)
+data class PdlPersonRequest<T>(
+    val variables: T,
+    val query: String
+)
```

Comment:
Dette oppsettet bryter med hvordan vi har satt opp styles i dag.

Reply:
Hvis jeg kjører reformat i intellij så gjøres det ikke noe med denne? Ser vi som følge av det har litt forskjellige måter å skrive data-classes på. Er det ikke bedre å ha en linter som gjør det likt?

Reply:
Nei, reformat vil aldri trekke noe en linje opp. Men hvis du flytter `val variables: T,` opp til linje 3 og deretter kjører reformat så tipper jeg den beholder den på linje 3. 

Jeg er litt bekymret for at vi begynner med ktlint her, `familie-settings` andre steder og får litt "hummer-og-kanari". Må vi kjøre en maven-kommando for å reformatere eller kan man få til dette direkte i intellij med en shortcut? Jeg er glad i sistnevnte for vanlig bruk og fint med en maven-kommando i prehook eller noe tilsvarende.

Reply:
Det stemmer, hvis `val variables: T` er flyttet opp til linje 3 så blir den beholdt der. Poenget er bare at jeg ser at det nå blir litt "hummer-og-kanari" mellom prosjektene fordi reformat ikke endrer dette selv om man skriver det ulikt. 

Selvfølgelig, vi må bli enige om stil og bruke dette i alle prosjektene. Dette er kun ment som pilot/poc (kjekt å starte med dette repoet, da det fortsatt er lite). Ja, man kan kjøre en maven kommando og binde det til shortcut. Kommandoen er `mvn ktlint:format` for dette. I pom'n har jeg lagt til slik at linter kjøres ved en `mvn install`. Jeg tenker at det er en god rutine å kjøre `mvn clean install` før man sjekker inn noe uansett. Jeg har også lagt til en pre-commit hook (pre-commit.sh) som kjører linter i denne PR'n.

Reply:
`familie-settings` bestemmer egentlige ikke noe om det er ny rad eller ikke etter `(`, men jeg synes egentlige det er mest riktig att det skal bli en ny rad. Hvis ikke så får du kode som går sikksakk som får øyene til å blø 
![image](https://user-images.githubusercontent.com/937168/145389982-b499c146-6d02-435c-80a6-5372cc28efa3.png)

Reply:
Skjønner. Jeg synes det er fint med lintern, men jeg ser andre ganger at det ville vært finest at det ikke ble ny rad. Blir nok ikke helt perfekt, men jeg synes det overall blir litt bedre når det blir likt overalt.

Reply:
Resultatet med linter: 
![Screen Shot 2021-12-09 at 13 22 40](https://user-images.githubusercontent.com/42737566/145395888-912ae517-ab12-4656-a70c-d46524d43acb.png)